vochain/process.go

```diff
@@ -143,8 +146,30 @@ func (v *State) CountProcesses(committed bool) (uint64, error) {
 	return count, nil
 }
 
+// ListProcessIDs returns the full list of process identifiers (pid).
+func (v *State) ListProcessIDs(committed bool) ([][]byte, error) {
+	if !committed {
+		v.Tx.RLock()
+		defer v.Tx.RUnlock()
+	}
+	processesTree, err := v.mainTreeViewer(committed).SubTree(StateTreeCfg(TreeProcess))
+	if err != nil {
+		return nil, err
+	}
+	var pids [][]byte
+	if err := processesTree.Iterate(func(key []byte, value []byte) bool {
+		p := make([]byte, len(key))
+		copy(p, key)
+		pids = append(pids, p)
+		return false
```

Comment:
doesn't this mean that we only iterate once and then stop? as in, `false` means we want to stop iterating.

Reply:
usually yes but https://github.com/vocdoni/vocdoni-node/blob/master/statedb/treeview.go#L42

Reply:
Huh, that's a bit weird. We should probably have a TODO to invert that.